src/pynwb/ecephys.py

```diff
@@ -108,7 +108,7 @@ class SpikeEventSeries(ElectricalSeries):
     electrode).
     """
 
-    __nwbfields__ = ()
+    __nwbfields__ = ({'name': 'unit_series', 'doc': 'doc', 'child': False},)
```

Comment:
`'doc'` needs to be something informative.

Reply:
Yes, but why is this throwing an error?